src/js/node/http.ts

```diff
@@ -1268,7 +1269,7 @@ ServerResponse.prototype.assignSocket = function (socket) {
     throw ERR_HTTP_SOCKET_ASSIGNED();
   }
   socket._httpMessage = this;
-  socket.on("close", onServerResponseClose);
+  socket.on("close", () => onServerResponseClose.$call(socket));
```

Comment:
theres never an error event or anything passed along as arguments to socket.on("close"), right?

Reply:
correct